### PR TITLE
Add instructions for Producer Portal

### PR DESCRIPTION
**Pull request by gibbleyg**

### CODE REVIEW

File: docs/deployment-manager-guide.md [hunk 1]
```diff
@@ -24,3 +24,28 @@ uncheck the **Metadata selection** box and then click **Continue**.
...
+
+```
+BUCKET_NAME=<BUCKET_NAME>
+gsutil mb $BUCKET_NAME
+gsutil versioning set on gs://$BUCKET_NAME
+gsutil iam ch "group:marketplace-ops@cloud.google.com:objectViewer" $BUCKET_NAME
+```
+
+Prior to running `mpdev apply`, ensure the `zipFilePath` specified
+in your [`DeploymentManagerTemplate`](../examples/deployment-manager/autogen/singlevm/configurations.yaml)
```
Comment (huyhg) [inline docs/deployment-manager-guide.md:41]: Add the `mpdev pkg set` command--reading this comment sorta implies editing the yaml file directly.

**Review by huyhg — APPROVED** [review]

File: docs/deployment-manager-guide.md (hunk 1 above)
Comment (gibbleyg) [reply docs/deployment-manager-guide.md:41]: Done

File: examples/deployment-manager/autogen/multivm/README.md [hunk 2]
```diff
@@ -74,5 +75,10 @@ for explanations of the fields.
 ## Upload Solution To Partner Portal
 
 See instructions in this 
-[guide](../../../../docs/deployment-manager-guide.md).
+[guide](../../../../docs/deployment-manager-guide.md#upload-solution-to-partner-portal).
+
+## Upload Solution to Producer Portal (Public Preview)
```
Comment (allisongh) [inline examples/deployment-manager/autogen/multivm/README.md:80]:
```suggestion
## Upload solution to Producer Portal
```

File: examples/deployment-manager/autogen/singlevm/README.md [hunk 3]
```diff
@@ -72,4 +73,9 @@ for explanations of the fields.
 ## Upload Solution To Partner Portal
 
 See instructions in this 
-[guide](../../../../docs/deployment-manager-guide.md).
+[guide](../../../../docs/deployment-manager-guide.md#upload-solution-to-partner-portal).
+
+## Upload Solution to Producer Portal (Public Preview)
+
```
Comment (allisongh) [inline examples/deployment-manager/autogen/singlevm/README.md:79]:
```suggestion
**Preview**: This product or feature is covered by the [Pre-GA Offerings Terms](https://cloud.devsite.corp.google.com/terms/service-terms#1) of the Google Cloud Platform Terms of Service. Pre-GA products and features may have limited support, and changes to pre-GA products and features may not be compatible with other pre-GA versions. For more information, see the [launch stage descriptions](https://cloud.devsite.corp.google.com/products#product-launch-stages).
```

File: examples/deployment-manager/autogen/singlevm/README.md [hunk 4]
```diff
@@ -72,4 +73,9 @@ for explanations of the fields.
 ## Upload Solution To Partner Portal
 
 See instructions in this 
-[guide](../../../../docs/deployment-manager-guide.md).
+[guide](../../../../docs/deployment-manager-guide.md#upload-solution-to-partner-portal).
+
+## Upload Solution to Producer Portal (Public Preview)
```
Comment (allisongh) [inline examples/deployment-manager/autogen/singlevm/README.md:78]:
```suggestion
## Upload solution to Producer Portal
```

File: examples/deployment-manager/autogen/singlevm/README.md [hunk 5]
```diff
@@ -72,4 +73,9 @@ for explanations of the fields.
 ## Upload Solution To Partner Portal
```
Comment (allisongh) [inline examples/deployment-manager/autogen/singlevm/README.md:73]:
```suggestion
## Upload solution to Partner Portal
```

File: examples/deployment-manager/autogen/multivm/README.md [hunk 6]
```diff
@@ -74,5 +75,10 @@ for explanations of the fields.
 ## Upload Solution To Partner Portal
```
Comment (allisongh) [inline examples/deployment-manager/autogen/multivm/README.md:75]:
```suggestion
## Upload solution to Partner Portal
```

File: docs/deployment-manager-guide.md [hunk 7]
```diff
@@ -24,3 +24,34 @@ uncheck the **Metadata selection** box and then click **Continue**.
...
+resource is pointing to the GCS bucket created above.
+
+If following an [example](../examples/deployment-manager/autogen/singlevm) in
+this repo, then the `zipFilePath` can be set programmatically with:
+```
+mpdev cfg set <DIR> zipPath gs://$BUCKET_NAME/dm-template.zip
+```
+
+After running `mpdev apply`, open
```
Comment (allisongh) [inline docs/deployment-manager-guide.md:50]:
```suggestion
After you run `mpdev apply`, upload the deployment package to Producer Portal:

1. Open [Producer Portal](https://console.cloud.google.com/producer-portal) in the Google Cloud Console.
1.  In the list of products, click the name of your product.
1. On the **Overview** page for your product, go to the **Deployment Package**
 section and click **EDIT**.
1. Under **Specify your GCS object location**, select the deployment package
 object that you uploaded to Google Cloud Storage.
```

File: docs/deployment-manager-guide.md [hunk 8]
```diff
@@ -24,3 +24,34 @@ uncheck the **Metadata selection** box and then click **Continue**.
 **WARNING:** Unchecking the **Metadata selection** box is crucial, so that
 `mpdev` does not override the solution metadata configured in earlier steps of
 the Partner Portal wizard.
+
+## Upload solution to Producer Portal (Public Preview)
```
Comment (allisongh) [inline docs/deployment-manager-guide.md:28]:
```suggestion
## Upload solution to Producer Portal
```

**Review by allisongh — CHANGES_REQUESTED** [review]
Thanks for these updates! Left a few suggested changes.

File: docs/deployment-manager-guide.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (gibbleyg) [reply docs/deployment-manager-guide.md]: Fixed

**Pull request merged**